src/components/pages/Map.tsx

```diff
@@ -54,6 +85,28 @@ function Map() {
   // },[]);
   // </座標取得 未実装>
 
+  // <表示範囲判定>
+  // const [selected, setSelected] = useState(null);
+  const [bounds, setBounds] = useState({ lat: 0, lng: 0 });
+
+  const mapRef = React.useRef();
+
+  const onMapLoad = React.useCallback((map) => {
```

Comment:
引数で渡しているmapの部分でエラーが発生しております。
型付け関連のエラーかと思われますが、解消方法が分からず困っております。

Reply:
GoogleMap コンポーネントの onLoad プロップスに onMapLoad を渡していると思いますが、onLoad プロップスの型が `onLoad?: ((map: google.maps.Map) => void | Promise<void>) | undefined` になっているので、こんな感じでしょうか？
```suggestion
  const onMapLoad = React.useCallback((map: google.maps.Map) => {
```